crates/ursa-gateway/Cargo.toml

```diff
@@ -11,6 +11,8 @@ description = "Ursa's proxy gateway"
 anyhow.workspace = true
 axum.workspace = true
 axum-server.workspace = true
+bincode.workspace = true
```

Comment:
remove these dependencies if not using them. I assume this was added when metadata was being decoded